mne/preprocessing/maxwell.py

```diff
@@ -815,33 +805,26 @@ def _sss_basis_point(origin, info, int_order, ext_order, imbalance,
                          'magnetometers. Currently have %i' %
                          imbalance.shape[1])
 
-    # Initialize internal multipolar moment space for point-like magnetometers
-    S_in_all = np.zeros((len(info['chs']), get_num_moments(int_order, 0)))
-    S_out_all = np.zeros((len(info['chs']), get_num_moments(0, ext_order)))
-
     # Coil_type values for x, y, z point magnetometers
     pt_types = [FIFF.FIFFV_COIL_POINT_MAGNETOMETER_X,
                 FIFF.FIFFV_COIL_POINT_MAGNETOMETER_Y,
                 FIFF.FIFFV_COIL_POINT_MAGNETOMETER]
 
     # Loop over all coordinate directions desired and create point mags
+    S_tot = None
     for dir_ind in range(imbalance.shape[1]):
-
         temp_info = deepcopy(info)
         for ch in temp_info['chs']:
             ch['coil_type'] = pt_types[dir_ind]
         coils_add = _prep_meg_channels(temp_info, accurate=True,
-                                       elekta_defs=True,
-                                       head_frame=head_frame)[0]
-
-        S_in_add, S_out_add = _sss_basis(origin, coils_add, int_order,
-                                         ext_order)
-        S_in_all += S_in_add
-        S_out_all += S_out_add
-
-    # Scale spaces by gradiometer imbalance
-    S_in_all *= imbalance[:, [dir_ind]]
-    S_out_all *= imbalance[:, [dir_ind]]
```

Comment:
I'm pretty sure these should have been indented, since they depend on the looping index `dir_ind`?